src/clipper/clipper.cpp

```diff
@@ -4601,7 +4602,7 @@ void OffsetPaths(const Paths &in_polys, Paths &out_polys,
   double delta, JoinType jointype, EndType_ endtype, double limit)
 {
   ClipperOffset co(limit, limit);
-  co.AddPaths(in_polys, jointype, (EndType)endtype); 
+  co.AddPaths(in_polys, jointype, (EndType)endtype);
```

Comment:
Is this file entirely whitespace changes? Can you change your editor settings to modify whitespace only on lines that actually changed, or separate whitespace-only changes into a separate commit?